internal/btf/btf.go

```diff
@@ -454,8 +454,8 @@ func ProgramLineInfos(s *Program) (recordSize uint32, bytes []byte, err error) {
 // IsNotSupported returns true if the error indicates that the kernel
 // doesn't support BTF.
 func IsNotSupported(err error) bool {
-	ufe, ok := errors.Cause(err).(*internal.UnsupportedFeatureError)
-	return ok && ufe.Name == "BTF"
+	var ufe *internal.UnsupportedFeatureError
+	return xerrors.As(err, &ufe) && (err).(*internal.UnsupportedFeatureError).Name == "BTF"
```

Comment:
If `As` returns true, you can use `ufe` like so:
```suggestion
	return xerrors.As(err, &ufe) && ufe.Name == "BTF"
```